pagopa/io-dev-api-server
# [IOBP-432] Add wallet payment authorization WebView

⚠️ Depends on https://github.com/pagopa/io-dev-api-server/pull/322

## Short description
This PR adds the `iowallet` protocol into payment authorization webview outcome url.

## How to test
Start the `io-dev-api-server`, start the payment flow from the Wallet Playground in the IO app and check that the authorization webview handles the outcome url.
